src/canvas/SelectableCanvas.ts

```diff
@@ -656,6 +649,18 @@ export class SelectableCanvas<
     );
   }
 
+  setTargetFindTolerance(value: number) {
+    this.targetFindTolerance = value;
+    const size = Math.ceil(
+      2 * Math.max(this.targetFindTolerance, 1) * this.getRetinaScaling()
+    );
+    if (this.pixelFindCanvasEl.width < size) {
+      this.pixelFindCanvasEl.width = this.pixelFindCanvasEl.height = size;
+      const retina = this.getRetinaScaling();
+      this.pixelFindContext.scale(retina, retina);
+    }
```

Comment:
after playing around with this branch a bit and the feature, i think this should be slightly different:
```suggestion
  setTargetFindTolerance(value: number) {
    value = Math.floor(value); // floats aren't supported. but just in case get rid of them
    this.targetFindTolerance = value;
    const size = 2 * value + 1;
    this.pixelFindCanvasEl.width = this.pixelFindCanvasEl.height = size;
  }
```
notice that size is  2 * tolerance + 1. 1 is the pixel that we are hoverin and tolerance 3 for example, would mean we always take 3 pixels each side in addition to the original pixel.
So the pixelTargetFind canvas is always an odd number of pixels ( 1,3,5,7... )

Reply:
sounds correct